pkgs/development/python-modules/cryptolyzer/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, certvalidator
+, attrs
+, six
+, urllib3
+, cryptoparser
+, requests
+}:
+
+buildPythonPackage rec {
+  pname = "cryptolyzer";
+  version = "0.8.1";
+
+  src = fetchPypi {
+    pname = "CryptoLyzer";
+    inherit version;
+    sha256 = "sha256-FbxSjKxhzlpj3IezuLCQvoeZMG1q+OE/yn5vB/XE1rI=";
+  };
+
+  propagatedBuildInputs = [
+    certvalidator
+    attrs
+    six
+    urllib3
+    cryptoparser
+    requests
+  ];
+
+  doCheck = false; # Tests require networking
```

Comment:
Could still add a `pythonImportsCheck`.

Reply:
What do we gain from that?

Reply:
We avoid the case where we end up with a package that we can't even import successfully. It happens.